backend/core/views.py

```
@@ -3869,6 +3869,24 @@ def post(self, request, *args, **kwargs):
         return Response(status=status.HTTP_400_BAD_REQUEST)
 
 
+class QuickStartView(APIView):
+    serializer_class = QuickStartSerializer
```

Ruff (0.8.2)

`QuickStartSerializer` may be undefined, or defined from star imports

(F405)